packages/material-ui-lab/src/Rating/Rating.js

```diff
@@ -42,7 +42,6 @@ export const styles = (theme) => ({
   /* Styles applied to the root element. */
   root: {
     display: 'inline-flex',
-    position: 'relative',
```

Comment:
It's for this style:

<img width="151" alt="Capture d’écran 2020-09-12 à 13 09 01" src="https://user-images.githubusercontent.com/3165635/92994258-23a0db80-f4f9-11ea-8d24-cfa25acfb8b8.png">

Reply:
That's focus if the value is zero?

Reply:
Without:

<img width="393" alt="Capture d’écran 2020-09-12 à 13 14 58" src="https://user-images.githubusercontent.com/3165635/92994378-fbfe4300-f4f9-11ea-8d13-7ebfa3633fe1.png">

Reply:
> That's focus if the value is zero?

The radio with value=""

Reply:
We need the relative position because we position `.pristine input:focus + &` `absolute`, right? Should add a comment so that we know when we can remove it.

Reply:
You are right, happy to add a comment around "why?".